.changeset/spicy-masks-sort.md

```diff
@@ -3,4 +3,4 @@
 "@firebase/analytics": patch 
 ---
 
-Added Browser Extension check for Firebase Analytics. analytics.isSupported() will now return Promise<false> for extension environments. 
+Added Browser Extension check for Firebase Analytics. analytics.isSupported() will now return `Promise<false>` for extension environments. 
```

Comment:
Well, might as well backtick `analytics.isSupported()` too while you're in there.

Reply:
Good idea. done.